frontend/spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -56,10 +56,28 @@
         order.update_column(:state, "address")
       end
 
-      it "should associate the order with a user" do
-        order.update_column :user_id, nil
-        expect(order).to receive(:associate_user!).with(user)
-        get :edit
+      context "current behavior" do
```

Comment:
It might be better if we say `"with associate_user_in_authentication_extension enabled"` as this behaviour will change with the version bump.